src/ResourceManager/Insights/Commands.Insights/Alerts/AddAzureRmAlertRuleCommandBase.cs

```diff
@@ -77,11 +77,11 @@ public abstract class AddAzureRmAlertRuleCommandBase : ManagementCmdletBase
         /// </summary>
         protected override void ProcessRecordInternal()
         {
-            WriteWarning("This output of this cmdlet will change in the next release to return the updated or newly created object.");
+            WriteWarning("The output of this cmdlet will change in the next release to return a single object, not a list, that includes the updated or newly created object.");
```

Comment:
Making the change for this and AddAzureRmAutoscaleSetting command too (also announced months ago).